react/src/components/Chat/ChatTokenCounter.tsx

```diff
@@ -0,0 +1,63 @@
+import { useTokenCount } from '../../hooks/useTokenizer';
+import Flex from '../Flex';
+import type { Message } from '@ai-sdk/ui-utils';
```

Comment:
We can refer it form `ai`.
```suggestion
import { Message } from 'ai';
```